labs/src/main/scala/org/scalalabs/basic/lab02/CollectionExercise.scala

```diff
@@ -64,9 +82,16 @@ object CollectionExercise03 {
    * checkValuesIncrease(Seq(1,2,3)) == true
    * checkValuesIncrease(Seq(1,2,2)) == false
    */
-  def checkValuesIncrease[T <% Ordered[T]](seq: Seq[T]): Boolean =
-    error("fix me")
-
+  def checkValuesIncrease[T <% Ordered[T]](seq: Seq[T]): Boolean = {
+    var preVal: Int = 0
+    for(val <- seq) {
+      if(val < preVal) {
+        false
+      }
+      preVal = val
+    }
+  true
```

Comment:
This is an interesting one. There are good functional ways to do it, but I don't find the for loop to obtrusive.

In particular, I wonder which functional methods allow for early exit?

(Also, one functional way begins with zipping together each pair of consecutive values. What happens next?)